Classes/WPEditorView.m

```diff
@@ -1290,6 +1304,26 @@ - (void)insertHTML:(NSString *)html
 
 #pragma mark - Editing
 
+- (void)wrapSourceViewSelectionWithTag:(NSString *)tag
```

Comment:
We could assert that tag is not `nil`.  Not a big addition but assertions can work wonders with cases such as this one when something goes bad.